src/pkg/bundle/remote.go

```diff
@@ -336,3 +336,33 @@ func EnsureOCIPrefix(source string) string {
 	}
 	return source
 }
+
+// get zarf package name mappings from oci provider
```

Comment:
nit: need to include `ZarfPackageNameMap` in the function's comment (our linter should really be catching this stuff)